modules/apps/portal-store/portal-store-gcs/src/main/java/com/liferay/portal/store/gcs/GCSStore.java

```diff
@@ -49,12 +50,12 @@
 
 import java.nio.channels.Channels;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Stream;
-import java.util.stream.StreamSupport;
+import java.util.Spliterator;
```

Comment:
@jeffwu0724 Why Spliterator is needed here?

Reply:
@tinatian When I try to remove the usage of `Stream<Blob> blobStream = StreamSupport.stream(blobs.spliterator(), false);`, StreamSupport declares a blobs.spliterator(). Therefore, I also try to use a `spliterator.forEachRemaining()` to iterate it and add to the new List.